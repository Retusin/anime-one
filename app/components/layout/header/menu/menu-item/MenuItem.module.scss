.menu_item {
  @apply text-[16px] cursor-pointer hover:text-violet-400 transition-colors duration-200 font-light;
}
