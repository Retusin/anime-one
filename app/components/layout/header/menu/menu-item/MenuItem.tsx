import Link from 'next/link';
import { FC } from 'react';

import { MenuDataParams } from '../menu.data';

import styles from './MenuItem.module.scss';

interface MenuItemParams {
  item: MenuDataParams;
}

export const MenuItem: FC<MenuItemParams> = ({ item }) => (
  <li>
    <Link href={item.link}>
      <span className={styles.menu_item}>{item.title}</span>
    </Link>
  </li>
);
