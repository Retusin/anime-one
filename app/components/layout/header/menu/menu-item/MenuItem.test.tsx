import { MenuItem } from '@/components/layout/header/menu/menu-item/MenuItem';
import { render } from '@testing-library/react';

test('render menu', () => {
  const { container } = render(
    <MenuItem item={{ link: '/home', title: 'Home' }} />
  );
  expect(container).toMatchSnapshot();
});
