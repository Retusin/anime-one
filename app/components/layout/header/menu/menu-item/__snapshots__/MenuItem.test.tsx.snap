// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render menu 1`] = `
<div>
  <li>
    <span
      class="menu_item"
    >
      Home
    </span>
  </li>
</div>
`;
